api/v1alpha1/obclusteroperation_types.go

```diff
@@ -32,24 +35,32 @@ type OBClusterOperationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	OBCluster          string                        `json:"obcluster"`
-	Type               apitypes.ClusterOperationType `json:"type"`
-	Force              bool                          `json:"force,omitempty"`
-	AddZones           []apitypes.OBZoneTopology     `json:"addZones,omitempty"`
-	DeleteZones        []string                      `json:"deleteZones,omitempty"`
-	AdjustReplicas     []AlterZoneReplicas           `json:"adjustReplicas,omitempty"`
-	RestartOBServers   *RestartOBServersConfig       `json:"restartOBServers,omitempty"`
-	Upgrade            *UpgradeConfig                `json:"upgrade,omitempty"`
-	ModifyStorageClass *ModifyStorageClassConfig     `json:"modifyStorageClass,omitempty"`
-	ExpandStorageSize  *ExpandStorageSizeConfig      `json:"expandStorageSize,omitempty"`
-	SetParameters      []apitypes.Parameter          `json:"setParameters,omitempty"`
+	OBCluster string                        `json:"obcluster"`
+	Type      apitypes.ClusterOperationType `json:"type"`
+	Force     bool                          `json:"force,omitempty"`
+	//+kubebuilder:default="7d"
+	TTL              string                    `json:"ttl,omitempty"`
```

Comment:
does ttl only support xxxd

Reply:
Yes, the same as backup `jobKeepWindow`

Reply:
Maybe it's better to use TTLDays and an int value

Reply:
OK